src/components/FooterTabs.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Switch, Route, Link } from 'react-router-dom';
+import ShoppingList from '../pages/ShoppingList';
+import AddItem from '../pages/AddItem';
+import '../css/FooterTabs.css';
+
+export default function App(props) {
+	return (
+		<div>
+			<nav>
+				<ul>
+					<li>
+						{/* Link to be changed once pages are created*/}
+						<Link to='/ShoppingList'>
+							<button className='navbutton'>Shopping List</button>
+						</Link>
+					</li>
+					<li>
+						<Link to='/AddItem'>
+							<button className='navbutton'>Add Item</button>
+						</Link>
+					</li>
+				</ul>
+			</nav>
```

Comment:
Here, too...nice.

Reply:
These are accessible via keyboard which is great. I think in the future we could eliminate the nested button inside the Link. I think the styling on these looks really sharp.